source/tutorials/fmt-cdiff-khanna.rst

```diff
@@ -400,6 +400,87 @@ For more information about animated ordinations, visit Emperor's `documentation`
+    --m-metadata-file sample_metadata_select.tsv \
+    --m-metadata-file core-metrics-results/unweighted_unifrac_first_distances.qza \
+    --m-metadata-file core-metrics-results/observed_otus_vector.qza \
+    --m-metadata-file core-metrics-results/unweighted_unifrac_pcoa_results.qza \
+    --p-state-column timepoint --p-default-group-column ibd_or_not \
+    --p-default-metric Distance \
+    --p-individual-id-column subject_code \
+    --o-visualization volatility.qzv
+
+
+In the interactive visualization, use the "Metric column" widget found in the toolbar on the right-hand side of the line plot to toggle between metrics displayed on the y-axis. Focus on "Distance" (UniFrac distance from donor), "observed_otus", and "Axis 1" (PCoA axis 1 ordinations). From these metrics, we see that 1) each subject's stool microbiota become more similar to the donor's after FMT; 2) FMT increases observed richness; and 3) FMT causes all samples to shift along PC axis 1, indicating a common shift in the microbial composition of those samples. We can test the significance of all of these effects using q2-longitudinal.
+
+The effects of FMT do not appear to be gradual, so we will bypass other approaches in q2-longitudinal for repeated measurements (see the :doc:`q2-longitudinal tutorial <longitudinal>` for more details on other methods in this plugin). Instead, we will focus just on changes in the microbiome pre- and post-FMT. We will use the ``pairwise-differences`` visualizer to perform Wilcoxon signed-rank tests of paired differences between each subject's samples pre- and post-FMT:
+
+.. command-block::
+
+  qiime longitudinal pairwise-differences \
+    --m-metadata-file sample_metadata_select.tsv \
+    --m-metadata-file core-metrics-results/unweighted_unifrac_first_distances.qza \
+    --p-state-column day_since_fmt \
+    --p-state-1 -1 \
+    --p-state-2 7 \
+    --p-metric Distance \
+    --p-individual-id-column subject_code \
+    --o-visualization does-fmt-make-the-stool-microbiome-resemble-donor.qzv
+
+  qiime longitudinal pairwise-differences \
+    --m-metadata-file sample_metadata_select.tsv \
+    --m-metadata-file core-metrics-results/observed_otus_vector.qza \
+    --p-state-column day_since_fmt \
+    --p-state-1 -1 \
+    --p-state-2 7 \
+    --p-metric observed_otus \
+    --p-individual-id-column subject_code \
+    --o-visualization does-fmt-increase-alpha-diversity.qzv
+
+  qiime longitudinal pairwise-differences \
+    --m-metadata-file sample_metadata_select.tsv \
+    --m-metadata-file core-metrics-results/unweighted_unifrac_pcoa_results.qza \
+    --p-state-column day_since_fmt \
+    --p-state-1 -1 \
+    --p-state-2 7 \
+    --p-metric 'Axis 1' \
+    --p-individual-id-column subject_code \
+    --o-visualization does-fmt-cause-a-directional-shift-in-pc1.qzv
+
+So that's neat; using these tests we have confirmed that following FMT:
+
+1. Each subject's stool microbiota become more similar to the donor's.

```

Comment:
I'm not sure how to rephrase or if leaving is fine. My issue is that this could be read as the samples are the "same" as the samples from donors, while we generally say that the samples are in the donor space but not the same as each of the individuals have their own trajectory. @nbokulich, what do you think?

Reply:
I don't think "more similar" can be misconstrued as "the same".